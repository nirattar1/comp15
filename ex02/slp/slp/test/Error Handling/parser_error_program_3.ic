class Stam {
	void foo() {
		i = i+1;
	}
}

int foo2(int i, string s, int[] arr){}


class Stam2 extends Stam {}