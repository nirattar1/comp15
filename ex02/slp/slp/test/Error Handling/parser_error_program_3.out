7: Syntax error: at token int